packages/feedback-service/src/resolver.ts

```diff
@@ -21,7 +22,33 @@ export const FeedbackResolver = {
   Mutation: {
     addFeedback(root: any, args: any, ctx: any) {
       const data = new Feedback(args.input);
-      return data.save();
+      return data.save().then(response => {
+        Feedback.findById(response._id)
+        .then(fb => {
+          if (fb){
+            const issue = {
+              "fields":{
+                "project":{
+                  "key": process.env.PROJECT_KEY
+                },
+                "summary": fb.title,
+                "description": fb.description,
+                "issuetype": {"name": "Task"},
+              }
+            }
+            const envpath = process.env.NODE_ENV;
+            if(envpath === "production"){
+              addFeedback(issue).then((jira:any) => {
+                args.input.ticketID = jira.key
+                return Object.assign(fb, args.input).save().then((feedback: any) => { return feedback; });
```

Comment:
Use the model.update instead of .save here.

Reply:
you mean Feedback.update({_id:...}, $set{ticketID:jira.key}) ?

Reply:
No, just a simple `fb.update(updatedObject)` should be enough.
Or even `Feedback.updateOne(...)`.

Check the docs:
https://mongoosejs.com/docs/api.html#model_Model.updateOne
https://mongoosejs.com/docs/api/document.html#document_Document-update